src/Enterspeed.Source.UmbracoCms.Shared.NetCore/Enterspeed.Source.UmbracoCms.Shared/EnterspeedPropertyValidation/EnterspeedPropertyValueValidator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Enterspeed.Source.Sdk.Api.Models.Properties;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Enterspeed.Source.UmbracoCms.Shared.EnterspeedPropertyValidation
+{
+    public class EnterspeedPropertyValueValidator : IEnterspeedPropertyValueValidator
+    {
+        private bool Valid(IEnterspeedProperty property)
+        {
+            var propertyValue = (IEnterspeedPropertyValue)property;
+            return propertyValue?.Value != null;
+        }
+
+        /// <summary>
+        /// Call to validate Enterspeed property
+        /// </summary>
+        /// <param name="property"></param>
+        /// <returns>A single property validation response</returns>
+        public IEnterspeedValidationResponse Validate(IEnterspeedProperty property)
+        {
+            var isValid = Valid(property);
+
+            var validationMessage = isValid ? "valid" : $"It is not allowed to a assign null value to {property.GetType()}. Please make sure that {property.Name} has a value";
```

Comment:
Maybe I'm reading this wrong. But in the `Valid` method you are validating the value but here it says that the `Name` must have a value?

Reply:
Property has a Name and Value. We are checking if Value if null. 

It is basically to give the user and idea of which property that is missing a value. :) 
We could word it differently 

"Please make sure that property with {property.Name} is not null. "

Reply:
Ahh, my bad. It makes sense, I was just reading it wrong. It's fine as it is :-)